x-pack/filebeat/input/netflow/convert.go

```diff
@@ -24,13 +24,38 @@ import (
 )
 
 func toBeatEvent(flow record.Record, internalNetworks []string) (event beat.Event) {
+	var e beat.Event
 	switch flow.Type {
 	case record.Flow:
-		return flowToBeatEvent(flow, internalNetworks)
+		e = flowToBeatEvent(flow, internalNetworks)
 	case record.Options:
-		return optionsToBeatEvent(flow)
+		e = optionsToBeatEvent(flow)
 	default:
-		return toBeatEventCommon(flow)
+		e = toBeatEventCommon(flow)
+	}
+
+	normaliseIPFields(e.Fields)
+	return e
+}
+
+// normaliseIPFields normalises net.IP fields in the given map from []byte to string. Note that
+// this function mutates the given map and relies on the fact that every net.IP field is direct
+// entry in the map. net.IP fields that do not meet the former convention, such being part of a struct,
+// etc., are not normalised.
```

Comment:
(Totally unnecessary refactoring to simplify and use an active voice. 😓 Feel free to ignore. And it took all my restraint to not switch "-ise" to "ize".)
```suggestion
// normaliseIPFields normalises net.IP fields in the given map from []byte to string.
// This function mutates the map and assumes every net.IP field is a direct entry.
// Fields that don't adhere to this convention (e.g. part of a struct) are not
// normalised.
```